src/features/expenses/AddExpense/AddParticipants.tsx

```diff
@@ -24,11 +25,12 @@ import PercentageSplitMethodView from "./SplitMethods/Percentage";
 import { ICreateTransaction, SplitMethod } from "../types";
```

Comment:
Please use one style for strings. 'example' or "example" (p.s. 'example' preferable)